substrate/frame/election-provider-support/src/lib.rs

```diff
@@ -298,16 +311,21 @@ pub trait ElectionDataProvider {
 	///
 	/// This should be implemented as a self-weighing function. The implementor should register its
 	/// appropriate weight at the end of execution with the system pallet directly.
-	fn electable_targets(bounds: DataProviderBounds)
-		-> data_provider::Result<Vec<Self::AccountId>>;
+	fn electable_targets(
```

Comment:
Rustdoc is outdated. Finding a bit hard without some explanation of how these functions are supposed to be used.

For example: why not pass `page_index` instead of `remaining_pages`?